R/posterior_linpred.R

```diff
@@ -43,6 +43,14 @@
 #'   transformed) linear predictor. The exception is if the argument \code{XZ}
 #'   is set to \code{TRUE} (see the \code{XZ} argument description above).
 #'   
+#' @note For models estimated with \code{\link{stan_clogit}}, the number of 
+#'   successes per stratum is ostensibly fixed by the research design. Thus,
+#'   when calling \code{posterior_linpred} with new data and \code{transform =
+#'   TRUE}, the \code{data.frame} passed to the \code{newdata} argument must
+#'   contain an outcome variable and a stratifying factor, both with the same
+#'   name as in the original \code{data.frame}. Then, the probabilities will
```

Comment:
Currently the `strata` argument to `stan_clogit` can either be the (unquoted) name of a variable (which is then looked for in `data`) or an actual factor that doesn't have to correspond to a column in `data`. In the second case there is no variable name in the data frame for the strata variable so how will `newdata` work?

Reply:
I am going to prevent the case where `stratum` involves something outside the `data.frame`.